djstripe/models/core.py

```diff
@@ -466,7 +466,7 @@ def credits(self):
 		return abs(min(self.account_balance, 0))
 
 	@property
-	def payment_methods(self):
+	def customer_payment_methods(self):
```

Comment:
Mention this change in HISTORY.rst

Reply:
(this will require a minor version change)